docs/run-instance/running-with-docker.md

```diff
@@ -1,247 +1,509 @@
 
 ## Prerequisites
 
-* [Docker](https://docs.docker.com/get-docker/) should be installed on your system.
+[Docker](https://docs.docker.com/get-docker/) has to be installed on your system. 
+No java or maven required unless you want to build a customized docker image :)
+
 
 ## Running prebuilt images
 
-There are multiple ways with Docker to quickly have a running instance.
+The openrouteservice team provides a `docker compose` file that pulls and starts the prebuilt openrouteservice docker image for a specific version in one simple command.
+Starting with openrouteservice v8.0.0, you can (and should) download the compose file from the "assets" section 
+on the [releases page](https://github.com/GIScience/openrouteservice/releases), in the browser or e.g. with 
 
-1. Recommended: Run latest release version image using `docker compose`
+```shell
+wget https://github.com/GIScience/openrouteservice/releases/download/v8.0.0/docker-compose.yml
+```
 
-    ```shell
-    # For latest release build
-    wget https://raw.githubusercontent.com/GIScience/openrouteservice/main/docker-compose.yml
-    docker compose up -d
-    ```
-   This will pull the latest release build of openrouteservice from Docker Hub and start it up using an example setup
-   and the provided test OSM file for Heidelberg/Germany and surrounding area.
+Now start openrouteservice in the background:
+```shell
+docker compose up -d
+```
 
-2. Run specific version image using `docker compose`
+This will pull the openrouteservice of the selected version and start it up using an example setup
+and the provided test OSM file for Heidelberg/Germany and surrounding area.
 
-   You can specify the Docker image tag by declaring an environment variable named `ORS_TAG`. Possible values are
-   specific version tags such as `v8.0.0` or `nightly` for the newest nightly build.
+To see the container's logs, run
+```shell
+docker compose logs 
+```
+or 
+```shell
+docker compose logs -tf  
+```
+to follow the log.
 
-    ```shell
-    # For nightly builds
-    wget https://raw.githubusercontent.com/GIScience/openrouteservice/main/docker-compose.yml
-    export ORS_TAG=nightly && docker compose up -d
-    ```
+To stop the container again, execute
+```shell
+docker compose stop 
+```
+or
+```shell
+docker compose down
+```
+to stop and remove the container.
 
-## Build image
+The file `docker-compose.yml` contains plenty of commented options with descriptions. 
+You can un-comment those options and set your custom option values.
+
+::: details Running without docker compose
+It is easier to modify the Docker compose file, but to test a specific setup, you can also run a command similar to what docker compose would run directly on the command line. 
+Here is a basic example:
+```shell
+# create directories for volumes to mount as local user
+mkdir -p ors-docker/config ors-docker/elevation_cache ors-docker/graphs ors-docker/files ors-docker/logs 
+docker run -dt -u "${UID}:${GID}" \
+  --name ors-app \
```

Comment:
```suggestion
docker run -dt --name ors-app \
```

Reply:
Let's keep the UID and GID out of the examples. If the user has another UID/GID than 1000:1000 it'll cause problems.